perception/object_segmentation/elevation_map_loader/README.md

```diff
@@ -0,0 +1,112 @@
+| use_lane_filter                   | bool        | Whether to filter elevation_map with vector_map                                                            | false         |
+| lane_margin                       | float       | Value of how much to expand the range of vector_map [m]                                                    | 0.5           |
+| lane_height_diff_thresh           | float       | Only point clouds in the height range of this value from vector_map are used to generate elevation_map [m] | 1.0           |
+| lane_filter_voxel_size_x          | float       | Voxel size x for calculating point clouds in vector_map [m]                                                | 0.04          |
+| lane_filter_voxel_size_y          | float       | Voxel size y for calculating point clouds in vector_map [m]                                                | 0.04          |
+| lane_filter_voxel_size_z          | float       | Voxel size z for calculating point clouds in vector_map [m]                                                | 0.04          |
+
+#### GridMap parameters
+
+The parameters are described on `config/elevation_map_parameters.yaml`.
+
+##### General parameters
+
+| Name                                           | Type | Description                                                                                                  | Default value |
+| :--------------------------------------------- | :--- | :----------------------------------------------------------------------------------------------------------- | :------------ |
+| pcl_grid_map_extraction/num_processing_threads | int  | Number of threads for processing grid map cells. Filtering of the raw input point cloud is not parallelized. | 12            |
+
+##### Grid map parameters
+
+See: <https://github.com/ANYbotics/grid_map/tree/ros2/grid_map_pcl>
+
+Resulting grid map parameters.
+
+| Name                                                     | Type  | Description                                                                                                                                      | Default value |
+| :------------------------------------------------------- | :---- | :----------------------------------------------------------------------------------------------------------------------------------------------- | :------------ |
+| pcl_grid_map_extraction/grid_map/min_num_points_per_cell | int   | Minimum number of points in the point cloud that have to fall within any of the grid map cells. Otherwise the cell elevation will be set to NaN. | 3             |
+| pcl_grid_map_extraction/grid_map/resolution              | float | Resolution of the grid map. Width and length are computed automatically.                                                                         | 0.3           |
+
```

Comment:
Would you add `height_type` and `height_thresh` ?
Please reference following and note that ros2 grid_map has no `Largest value` method.
```
* **pcl_grid_map_extraction/grid_map/height_type** How to calculate elevation value. 
0: Smallest value among the average v3lues ​​of each cluster 
1: Mean value of the cluster with the most points
* **pcl_grid_map_extraction/grid_map/height_thresh** Height range from the smallest cluster.(For only height_type 1)
```
https://github.com/ANYbotics/grid_map/pull/329

Reply:
please refer to https://github.com/tier4/AutowareArchitectureProposal.iv/pull/740/commits/d2330c0aeb89389b526dd2d3cfb6d0b17a3440ad

Reply:
It is the height of the cluster, not the point.
Would you simply copy mine?

Reply:
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/740/commits/898da971fcd1aead155240407eb2354d63f5fe16

Reply:
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/740/commits/879e5e9eef6e1b59d0c362a5d318c1b861e3d0df

Reply:
OK, thanks.
And height_thresh works only for 1. Plase add like `(Only for height_type 2)`

Reply:
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/740/commits/fa83f3dea24a22e6175bae40206b603d33b5fd35